### ISSUE TITLE
question about mask dimension

### ISSUE THREAD

**Issue by spacewalk01** [NVIDIA-AI-IOT/nanosam#14]
Thank you for your work! The output mask shape is 4x256x256 where 4 is (I guess) the number of labels and 256x256 is mask's height x width dimensions. I wonder how to get 1x256x256 mask from the output?

**Comment by spacewalk01** [comment]
https://github.com/NVIDIA-AI-IOT/nanosam/blob/653633614b2eb93b06ba3be9adb2aeffb117bd72/nanosam/utils/predictor.py#L134C7-L134C7

**Comment by spacewalk01** [comment]
you are performing `(mask[0, 0] > 0).detach().cpu().numpy()`  on `1 x 4 x 256 x 256` matrix which means that first only 256 x 256 returns ignoring the rest,  is it background mask?

**Comment by spacewalk01** [comment]
I am reimplementing it but in my implementation, it gets me incorrect results on the mask detection task given a bounding box.
